PEA/Makefile

```diff
@@ -1,11 +1,14 @@
 .DEFAULT_GOAL := all
 
-java_path := $(abspath ../build/linux-x86_64-server-fastdebug/images/jdk)
-export JAVA_HOME := ${java_path}
+JAVA_PATH := $(abspath ../build/linux-x86_64-server-fastdebug/images/jdk)
+export JAVA_HOME := ${JAVA_PATH}
 export PATH := ${JAVA_HOME}/bin:$(PATH)
```

Comment:
We should avoid from using 'environment variable' here. 

In Makefile, we can always define gmake variables.  I understand that we have to define JAVA_HOME and PATH for auto.sh, we can do that like you did for ctw.sh with leading env JAVA_OPTIONS.

Reply:
Sorry, don't quite understand. I just moved the logic from auto.sh into Make. I don't think these environment variables are bad for CTW.